vsts/pipelines/buildimageBases.yml

```diff
@@ -0,0 +1,63 @@
+variables:
+  ascName: OryxMCR
```

Comment:
note: need to change the path for base image build definition once it's merged in master